server/controllers/appt.controller.ts

```diff
@@ -0,0 +1,54 @@
+import { Request, Response } from "express";
+import Appointment from "../models/appt.model";
+import { IAppointment } from "../interfaces/appointment.interface";
+
+const findAllAppointments = async (req: Request, res: Response): Promise<IAppointment[]> => {
+  try {
+    const appointments = await Appointment.find()
+    return res.status(200).json(appointments)
+  } catch(e) {
+    return res.status(500).json(e.message)
+  }
+}
+
+const createNewAppointment = async (req: Request, res: Response): Promise<IAppointment> => {
+  try {
+    const appointment = await Appointment.create(req.body)
+    return res.status(201).json(appointment)
+  } catch(e) {
+    return res.status(500).json(e.message)
+  }
+}
+
+const findOneAppointment = async (req: Request, res: Response): Promise<IAppointment> => {
+  try {
+    const appointment = await Appointment.findOne({ _id: req.params.id })
+    return res.status(200).json(appointment)
+  } catch(e) {
+    return res.status(500).json(e.message)
+  }
+}
+const updateAppointment = async (req: Request, res: Response): Promise<IAppointment> => {
+  try {
+    const appointment = await Appointment.findByIdAndUpdate({ _id: req.params.id }, req.body, { new: true })
+    return res.status(200).json(appointment)
+  } catch(e) {
+    return res.status(500).json(e.message)
+  }
+}
+const deleteAppointment = async (req: Request, res: Response): Promise<{}> => {
+  try {
+    const appointment = await Appointment.deleteOne({ _id: req.params.id })
+    return res.status(200).json(appointment)
+  } catch(e) {
+    return res.status(500).json(e.message)
+  }
+}
+
+export { 
```

Comment:
any reason to use this over bundling them all together in an `export default { }`? Either way works for me, just curious, as this'll be an extra step if we were to ever add any other functionality in..but that might be better.

Reply:
Resolved by @b-chang and @pmar0 post discord discussion.

There's a couple differences in syntax:

- `export` requires `import * as ApptController from <file>`
- `export default` requires `import ApptController from <file>`

We stuck with the latter, as it's what we're used to and don't see a use-case for importing anything less than the whole controller for the time being.